dash.js

```diff
@@ -527,6 +527,72 @@ var MyDash = GObject.registerClass({
             }
         });
 
+        let switchTimerId = { id: 0 };
+        let isDndTarget = false;
+        let scale = St.ThemeContext.get_for_stage(Meta.get_backend().get_stage()).scale_factor;
```

Comment:
You can get the sage via `global.stage`.

Reply:
Thanks.